### DISCUSSION TITLE
Change the server type from CPX21 to CPX31

### DISCUSSION THREAD

**Discussion by mabebrahimi** [kubermatic/kubeone#1946] [General]
I installed kube on Hetzner with examples
Now I want to change the server type from CPX21 to CPX31
What should I do?

**Comment by kron4eg** [comment]
@mabebrahimi hi! Do you mean control-plane nodes?

**Comment by mabebrahimi** [comment]
@kron4eg Both control-plane and worker-nodes

**Comment by xmudrii** [comment]
@mabebrahimi Scaling up worker nodes can be done by editing the MachineDeployment object and changing the type. You can do that by running `kubectl get machinedeployment -n kube-system`, and then edit each MachineDeployment.

Scaling up the control plane nodes is a bit more complicated. I recommend checking out https://github.com/kubermatic/kubeone/issues/1543#issuecomment-929402128 for more details.

**Comment by mabebrahimi** [comment]
@xmudrii Is there no way for scaling up worker nodes with terraform?
